packages/core/test/actions.test.ts

```diff
@@ -1023,7 +1078,10 @@ describe('entry/exit actions', () => {
       expect(() => interpreter.stop()).not.toThrow();
     });
 
-    it('sent events from exit handlers of a stopped child should not be received by the parent', () => {
+    // TODO: determine if the sendParent action should execute when the child actor is stopped.
+    // If it shouldn't be, we need to clarify whether exit actions in general should be executed on machine stop,
+    // since this is contradictory to other tests.
```

Comment:
I don't feel those are contradictory - exit actions are still executed, including this `sendParent`. The only difference is that the parent already knows that it has stopped this child, so it shouldn't receive any new events from it.

It doesn't matter if internally `sendParent` is actually executed or not. If it's prevented in the child or in the parent, the end result from the user's perspective is the same.

This is mentioned [here](https://www.w3.org/TR/scxml/#invokeimplementation):
> As described above, if the invoking state machine exits the state containing the invocation before it receives the done.invoke.id event, it cancels the invoked session. The method for doing this is platform-specific. However, when it is cancelled, the invoked session must exit at the end of the next microstep. The Processor must execute the <onexit> handlers for all active states in the invoked session, but it must not generate the done.invoke.id event. Once it cancels the invoked session, the Processor must ignore any events it receives from that session. In particular it must not not insert them into the external event queue of the invoking session.